setup.py

```diff
@@ -144,42 +152,66 @@ def run(self):
-MATLAB-like features in a simple and light-weighted software.
-It also provides ready-to-use pure-Python widgets to your PyQt5
-application: A source code editor with syntax highlighting and
-code introspection/analysis features, NumPy array editor, dictionary
-editor, Python console, etc.""",
-      download_url='%s/files/%s-%s.zip' % (__project_url__, NAME, __version__),
-      author="The Spyder Project Contributors",
-      url=__project_url__,
-      license='MIT',
-      keywords='PyQt5 editor shell console widgets IDE',
-      platforms=['any'],
-      packages=get_packages(),
-      package_data={LIBNAME: get_package_data(LIBNAME, EXTLIST),
-                    'spyder_breakpoints': get_package_data('spyder_breakpoints', EXTLIST),
-                    'spyder_profiler': get_package_data('spyder_profiler', EXTLIST),
-                    'spyder_pylint': get_package_data('spyder_pylint', EXTLIST),
-                    'spyder_io_dcm': get_package_data('spyder_io_dcm', EXTLIST),
-                    'spyder_io_hdf5': get_package_data('spyder_io_hdf5', EXTLIST),
-                    },
-      scripts=[osp.join('scripts', fname) for fname in SCRIPTS],
-      data_files=get_data_files(),
-      classifiers=['License :: OSI Approved :: MIT License',
-                   'Operating System :: MacOS',
-                   'Operating System :: Microsoft :: Windows',
-                   'Operating System :: POSIX :: Linux',
-                   'Programming Language :: Python :: 2.7',
-                   'Programming Language :: Python :: 3',
-                   'Development Status :: 5 - Production/Stable',
-                   'Topic :: Scientific/Engineering',
-                   'Topic :: Software Development :: Widget Sets'],
-      cmdclass=CMDCLASS)
+setup_args = dict(
+    name=NAME,
+    version=__version__,
+    description='The Scientific Python Development Environment',
+    long_description=(
+"""Spyder is a powerful scientific environment written in Python, for Python,
+and designed by and for scientists, engineers and data analysts.
+It features a unique combination of the advanced editing, analysis, debugging
+and profiling functionality of a comprehensive development tool with the data
+exploration, interactive execution, deep inspection and beautiful visualization
+capabilities of a scientific package.
+Furthermore, Spyder offers built-in integration with many popular
+scientific packages, including NumPy, SciPy, Pandas, IPython, QtConsole,
+Matplotlib, SymPy, and more.
+Beyond its many built-in features, Spyder's abilities can be extended even
+further via its plugin system and API.
+Spyder can also be used as a PyQt5 extension library, allowing you to build
+upon its functionality and embed its components, such as the interactive
+console or advanced editor, in your own software.
+"""),
+    download_url=__website_url__ + "#fh5co-download",
+    author="The Spyder Project Contributors",
+    author_email="ccordoba12@gmail.com",
```

Comment:
Yeah, good point. Removed as requested.

Reply:
we could use 

`admin@spyder-ide.org` ? I can set forwarding to your account @ccordoba12 ?

Reply:
It does seem most other big projects do have something there. Another alternative if you think we'll get a lot of people asking for help is how about our Google Group email spyderlib@googlegroups.com ? 

If we get a high percentage of help, development, etc queries through that email it would direct them to a better place where they can be resolved publicly; on the other hand there is a chance people might send private inquiries to it but being a Google Groups email it should be obvious that its public, and people who really care about confidentiality can easily just look you up on Github since your email is published right on your page.

Reply:
> admin@spyder-ide.org ? I can set forwarding to your account @ccordoba12 ?

But that's my point: people should understand that the best way to reach us is to open an issue on Github or start a conversation on Google Groups, *not* to write an email.

Reply:
> Another alternative if you think we'll get a lot of people asking for help is our Google Group email spyderlib@googlegroups.com

I agree with this one. @CAM-Gerlach, please use it.

Reply:
> I agree with this one. @CAM-Gerlach, please use it.

Done as requested.